src/utility/api.ts

```diff
@@ -0,0 +1,29 @@
+import axios from "axios";
+import { Task } from "../types/Task.types";
+
+export const getTasks = async (): Promise<Task[]> => {
+  const response = await axios.get(`${process.env.REACT_APP_API_HOST}/tasks`);
+  return response.data.data;
+};
+
+export const addTask = async (task: Task): Promise<Task> => {
+  const response = await axios.post(
+    `${process.env.REACT_APP_API_HOST}/tasks`,
```

Comment:
To make it more readable and since it's the same everywhere, I suggest extracting the URL in something like
```
const url = `${process.env.REACT_APP_API_HOST}/tasks`;
```
and replace it everywhere it's used